### ISSUE TITLE
Fails to load workflow with deprecated "pattern" trigger map

### ISSUE THREAD

**Issue by viktorbenei** [bitrise-io/bitrise-workflow-editor#36]
Trigger map:

```
trigger_map:
- workflow: _cleanup_docker_images
  pattern: cleanup/docker-images
- workflow: _cleanup_docker_containers
  pattern: cleanup/docker-containers
```

Results in:

![screen shot 2017-01-31 at 21 49 46](https://cloud.githubusercontent.com/assets/430618/22483885/5fe372ac-e7ff-11e6-87ae-6e67ac356ede.png)

**Comment by erosdome** [comment]
I managed to reproduce the bug, a fix is on its way and will be published in the next release

**Comment by viktorbenei** [comment]
Awesome, thanks @erosdome ;)

**Comment by erosdome** [comment]
Fixed with v0.9.3. release. More information:
https://discuss.bitrise.io/t/offline-workflow-editor-workflow-editor-v2-v0-9-3-released/650
